packages/DOS/README

```diff
@@ -1,7 +1,9 @@
 Gisle Vanem made curl build fine on DOS (and MingW) with djgpp, OpenSSL and his
 Watt-32 stack.
 
-'make djgpp' in the root curl dir should build it fine.
+'make -f Makefile.dist djgpp' in the root curl dir should build it fine.
+Or enter 'lib' and do a 'make -f Makefile.dj clean all' to first delete
+'lib/curl_config.h' which is possibly from a previous incompatible Windows-build.
```

Comment:
Does make djgpp work and if so why not just leave it as that

Reply:
> Does make djgpp work and if so why not just leave it as that

@gvanem can you address this

Reply:
@jay  You mean drop the changes to `README`?

Reply:
I mean can you address my only question outstanding, what's the difference between 'make djgpp' and 'make -f Makefile.dist djgpp', the former is simpler why not just leave it like that?

Reply:
A 'make djgpp' is not possible since there is no `Makefile` or `GNUMakefile` in root unless djgpp-user runs `configure`. Which they don't have to.
```
F:\MingW32\src\inet\curl>make djgpp
gnumake: *** No rule to make target 'djgpp'.  Stop.
```

Reply:
Besides the `clean` is important. When I do:
```
cmd /c buildconf.bat
make djgpp
```
this happens:
```
f:/MinGW32/src/gnu/GNU-Make/src/gnumake.exe -C lib -f Makefile.dj
gnumake[1]: Entering directory 'F:/MingW32/src/inet/curl/lib'
f:/gv/djgpp-7.2/bin/i586-pc-msdosdjgpp-gcc -g -O2 -I. -I../include -I../lib -Ie:/net/watt/inc -Wall -DHAVE_CONFIG_H -DBU
ILDING_LIBCURL -o djgpp/asyn-ares.o -c asyn-ares.c
In file included from curl_config.h:13:0,
                 from curl_setup.h:66,
                 from asyn-ares.c:23:
config-win32.h:723:10: fatal error: ws2tcpip.h: No such file or directory
 #include <ws2tcpip.h>
          ^~~~~~~~~~~~
compilation terminated.
gnumake[1]: *** [../packages/DOS/common.dj:191: djgpp/asyn-ares.o] Error 1
```

Try it and see for yourself.